README.md

```diff
@@ -195,11 +195,114 @@ Subscribe to the topic published by the AWS IoT thing in the client, to see mock
 ![mqtt subscription](images/mqtt_subscription.png)
 You can now convert any ROS2 topic data and send it over as an MQTT topic over AWS IoT Core, with a thing transformation layer from ROS2 topic to JSON-formatted messages sent over an MQTT topic.
 
+## IoT Shadow Node
```

Comment:
I'd add a corresponding IoT MQTT telemetry node section . Maybe right after [this line](https://github.com/hartmir/aws-iot-robot-connectivity-samples-ros2/blob/292ffe0acdf0ce9b063738e529fb163f625523c5/README.md?plain=1#L166), where we have "You have completed all the setup needed to send data to AWS IoT Core!"

Reply:
Agreed - coming in next commit